data/darktableconfig.xml.in

```diff
@@ -2318,4 +2318,11 @@
     <shortdescription>overwrite the screen's ppd setting</shortdescription>
     <longdescription>if this value is > 0.0 then it is used as the screen's ppd setting which is used for HiDPI support</longdescription>
   </dtconfig>
+  <dtconfig>
+    <name>plugins/lighttable/preview/full_size_preload_count</name>
+    <type>int</type>
+    <default>3</default>
```

Comment:
I'm on a train with a 1600x900 screen, not my 2560x1600 that I have at home, and therefore everything is much faster right now. The numbers which I had at home, however, were between 3 and 10 seconds for rendering a full-size preview with no preload. Whether a longer buffer helps or doesn't help is going to depend on both the processing speed (and therefore the number of active image operations), the time you spend with each image in the preview, and on other aspects of your workflow, for example whether you're previewing large continuous series or not. Furthermore, these previews are actually stored in the disk cache as well, so their improvement is "eternal" -- once they're generated, you're going to benefit from them forever, or at least until you open darkroom for them once again. That's why I disagree with you on the "as low as possible" evaluation, but I understand your point -- you probably do not browse through previews so heavily, so for you it's wasted CPU time, even though it only affects energy, not the user experience. Fair enough -- if I was only using the full preview every now and then, I wouldn't have a use for this preload, either.

So if it's going to help upstream this patch, I'll be fine with preloading just one image :).